src/views/Homepage/components/WelcomeContent/WelcomeContent.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
undo all changes to this file, we can access your page using the url path.

Reply:
Done